code/datums/gamemodes/TDM_tickets.dm

```diff
@@ -0,0 +1,73 @@
+/datum/game_mode/team_death_match
+	name = "Team Death Match"
+	config_tag = "TDM"
+	required_players = 0 //change to 2 in the future
```

Comment:
Warfare has 0 required players. Leaving it at 0 just makes testing easier. If you're going to play a game it's pretty much guaranteed you're going to have _at least_ 2 players.